cocos/core/assets/material.jsb.ts

```diff
@@ -88,7 +88,17 @@ function wrapSetProperty (cb: setProperyCB, target: Material, name: string, val:
         cb.call(target, name, val);
     }
 }
-
+enum  MathType {
```

Comment:
Remove a space between enum and MathType.

Reply:
1